libs/src/services/creatorNode/CreatorNodeSelection.test.js

```diff
@@ -300,7 +308,7 @@ describe('test CreatorNodeSelection', () => {
           latitude: '37.7058',
           longitude: '-122.4619'
         } })
-      services.push(healthyUrl)
+      contentNodes.push(healthyUrl)
     }
 
     let cns
```

Comment:
i'm unclear what this loop on L315 is doing - can you explain / comment? it looks like you're trying to test selections with a variable number of nodes from 5-1, and ensuring that returnedHealthyServices is always size 5

Reply:
yes! that's exactly it. I can add a comment